dev/PackageManager/API/PackageDeploymentResolver.cpp

```diff
@@ -179,12 +181,14 @@ winrt::hstring Microsoft::Windows::ApplicationModel::PackageDeploymentResolver::
     auto packages{ packageManager.FindPackagesForUserWithPackageTypes(winrt::hstring(), packageFamilyName, packageTypes) };
 
     // Find the/any match
-    auto criteria{ wil::str_printf<wil::unique_cotaskmem_string>(L"PackageFamilyName=%ls MinVersion=%hu.%hu.%hu.%hu ArchitectureFilter:0x%X",
-                                                                 packageFamilyName.c_str(),
-                                                                 minVersion.Major, minVersion.Minor,
-                                                                 minVersion.Build, minVersion.Revision,
-                                                                 static_cast<std::uint32_t>(processorArchitectureFilter)) };
-    Common::Logging::DebugLog(std::format(L"PackageDeploymentResolver: Scanning packages ({})", criteria.get()));
+    TraceLoggingWrite(
```

Comment:
Does TraceLoggingWrite write to both ETL and OutputDebugString?

Reply:
TraceLoggingWrite() writes to ETW

OutputDebugString() writes to ODS

WIL writes to both (technically WIL writes to ODS and an optional callback, but we register a callback in the usual way to wire up ETW support)

But for these Informational events we don't want ODS.

ETW is important for non-dev support scenarios but if you're a dev in VS you either don't care about this information, or if you really do you can debug into the source to see the actual code flow (equally or more meaningful than the previous LOG_HR to VS' Output window)

Reply:
Are we confident that nobody will miss this ODS output - that this is not a breaking change that people will mind?